Empire/Supply.cpp

```diff
@@ -163,6 +163,13 @@ bool SupplyManager::SystemHasFleetSupply(int system_id, int empire_id, bool incl
     return false;
 }
 
+bool SupplyManager::PlanetHasSupply(int planet_id) const {
```

Comment:
Since there are already a couple "SystemHasFleetSupply" methods here, and this is covering the same thing except at the planet level, it seems to me this should probably be named PlanetHasFleetSupply to be consistent.